os/fs/smartfs/smartfs_utils.c

```diff
@@ -588,7 +588,7 @@ off_t smartfs_seek_internal(struct smartfs_mountpt_s *fs, struct smartfs_ofile_s
 	 */
 
 	sectorstartpos = sf->filepos - (sf->curroffset - sizeof(struct smartfs_chain_header_s));
-	if (newpos >= sectorstartpos && newpos < sectorstartpos + fs->fs_llformat.availbytes - sizeof(struct smartfs_chain_header_s)) {
+	if ((newpos >= sectorstartpos) && (newpos < sectorstartpos + SMARTFS_AVAIL_DTATBYTES(fs))) {
```

Comment:
/root/tizenrt/os/../build/output/libraries/libfs.a(smartfs_smart.o): In function `smartfs_stat_common':
/root/tizenrt/os/fs/smartfs/smartfs_smart.c:1703: undefined reference to `SMARTFS_AVAIL_DTATBYTES'
/root/tizenrt/os/../build/output/libraries/libfs.a(smartfs_utils.o): In function `smartfs_seek_internal':
/root/tizenrt/os/fs/smartfs/smartfs_utils.c:591: undefined reference to `SMARTFS_AVAIL_DTATBYTES'
/root/tizenrt/os/fs/smartfs/smartfs_utils.c:613: undefined reference to `SMARTFS_AVAIL_DTATBYTES'
/root/tizenrt/os/../build/output/libraries/libfs.a(smartfs_utils.o): In function `smartfs_extendfile':
/root/tizenrt/os/fs/smartfs/smartfs_utils.c:2056: undefined reference to `SMARTFS_AVAIL_DTATBYTES'

please fix typo

Reply:
I already pushed to correct,
Let me check the issue, maybe version mistake

Reply:
Done